infrastructure/main.tf

```diff
@@ -52,6 +52,8 @@ module "api" {
 # region save DB details to Azure Key Vault
 module "key-vault" {
   source              = "git@github.com:hmcts/moj-module-key-vault?ref=master"
+  # Making sure Vault name will not exceed the maximum length of 24 charactes
+  name                = "${substr(format("%s-%s", var.product, var.env), 0, 24)}"
```

Comment:
what's the origin of this max length? can we not just increase it?

Reply:
netbios I believe ~30 years ago microsoft limitation ;)